test/check-machines-migrate

```diff
@@ -158,7 +158,7 @@ class TestMachinesMigration(VirtualMachinesCase):
 
         if fail == "uri":
             # test failure for syntactically incorrect destination uri
-            b.set_input_text("#dest-uri-input", "qemu+ssh://root@machine2/sistem")
+            b.set_input_text("#dest-uri-input", "qemu+ssh://root@machine2/system")
```

Comment:
Oh, is this intentionally a spelling error to fail? I guess I should back out this change.